flow360/component/simulation/unit_system.py

```diff
@@ -178,6 +178,8 @@ def _unit_object_parser(value, unyt_types: List[type]):
                     return unyt_type(value["value"], value["units"], dtype=np.float64)
                 except u.exceptions.UnitParseError:
                     pass
+                except RuntimeError:
```

Comment:
What is this for?

Reply:
when parsing scalar or vector - moment length type, unyt is not throwing error as expected (`u.exceptions.UnitParseError` but RuntimeError) and could infer type correctly.